DataLink.tex

```diff
@@ -437,20 +437,41 @@ \subsection{DataLink MIME Type}
 via the RESPONSEFORMAT parameter (see \ref{sec:responseformat}).
 
 \subsubsection{DataLink recognition outside the context
-               of DAL discovery services responses}
+               of ObsCore responses}
 \label{sec:columnsUrl}
 
-When providing a column with URLs, for example outside DAL service
-responses or when service descriptors  are not defined, if all the
-URLs are to a DataLink \blinks\ endpoint, then the preferred approach
-is to add a LINK element with the content type defined above.
+In order to provide access URL to \blinks\ endpoints outside the ObsCore-style
+service responses context  there are several options. 
+\begin{itemize}
+\item A LINK element SHOULD be added to the FIELD containing the string considered as the ID to be used in the DataLink service. The content-type of the LINK will be set to : 
+\begin{verbatim}
+   application/x-votable+xml;content=datalink
+\end{verbatim}
+and the href will be a template to the \blinks\ endpoint using the FIELD value, as described in the VOTable	 specification. 
```

Comment:
I consider  "as described in the VOTable specification" here is misleading; this templating business only appears in the "Possible VOTable Extensions" Appendix A of the VOTable spec, which is annotated _"The definitions enclosed in this appendix are not part of VOTable 1.1, but are considered as candidates for VOTable improvements."_. I don't know which if any VOTable parsing libraries have chosen to implement this templating mechanism, but STIL certainly doesn't.

Reply:
I removed that. See my answer to Markus above and to Pat below